skimage/data/_fetchers.py

```diff
@@ -1146,6 +1146,38 @@ def skin():
     return _load('data/skin.jpg')
 
 
+def nickel_solidification():
+    """Image sequence of synchrotron x-radiographs showing the rapid
+    solidification of a nickel alloy sample.
+
+    Returns
+    -------
+    nickel_solidification: (11, 384, 512) uint16 ndarray
+
+    Notes
+    -----
+    See https://gitlab.com/scikit-image/data/-/blob/master/README.md#data.
+
+    """
+    return _load('data/solidification.tif')
+
+
+def protein_transport():
+    """Microscopy image sequence with fluorescence tagging of proteins
+    re-localizing from the cytoplasmic area to the nuclear envelope.
+
+    Returns
+    -------
+    protein_transport: (15, 2, 180, 183) uint8 ndarray
+
+    Notes
+    -----
+    See https://gitlab.com/scikit-image/data/-/blob/master/README.md#data.
```

Comment:
👍🏻 This file has the license info

Reply:
Yes, exactly; I considered copying-pasting the licensing (and other) info from there, but let's maintain only one source and point to it.

But, your comment made me realize I should specify that `data/protein_transport.tif` corresponds to `NPCsingleNucleus.tif` at https://gitlab.com/scikit-image/data/-/blob/master/README.md#data. :+1: